README.md

```diff
@@ -13,3 +13,8 @@ The dates so far have been:
 
 The INTERSECT project is supported by NSF awards [2017424](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017424)
 and [2017259](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2017259).
+
+## Comments -- Acrivity
```

Comment:
Activity** but it's a comment so shouldn't really matter

Reply:
Thanks! Fixed on [3f0b907](https://github.com/jgarciamesa/intersect-colab-git/pull/2/commits/3f0b9077fc471c9ddbbe046f6527b7d4dd35057a).